docs/data-sources/images_image.md

```diff
@@ -74,6 +74,9 @@ data "huaweicloud_images_image" "bms_image" {
 
 * `enterprise_project_id` - (Optional, String) Specifies the enterprise project ID of the image.
 
+* `flavor_id` - (Optional, String) Specifies the ECS flavor ID used to filter out available public image. You can
+  specify only one flavor ID. You can specify only an ECS flavor ID and cannot specify a BMS flavor ID.
```

Comment:
1、Change "used to" to "is used to".
2、How about "Only support ECS flavor ID." replace "You can specify only an ECS flavor ID and cannot specify a BMS flavor ID."